fips/views.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .models import FipsEntity
+from .serializers import FipsSerializer
+
+
+@api_view(['GET'])
+def search_view(request):
+    """
+    Search for a location.
+    ---
+    parameters:
+      - name: q
+        description: The user's search query
```

Comment:
This get's published in the API docs, so it's worth going the extra mile to make sure these are descriptive. How about:

> The user's search query against jurisdiction names.